Marlin/Marlin_main.cpp

```diff
@@ -5441,7 +5500,15 @@ void home_all_axes() { gcode_G28(true); }
             #if ENABLED(PROBE_MANUALLY)
               z_at_pt[0] += lcd_probe_pt(cos(a) * r, sin(a) * r);
             #else
-              z_at_pt[0] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1, false);
+              z_at_pt[0] += probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1);
```

Comment:
It's ok to drop `false` argument, assuming we want to skip limiting to the printable area.